internal/cmd/util/validation.go

```diff
@@ -0,0 +1,44 @@
+package util
+
+import (
+	"fmt"
+	"os"
+	"regexp"
+	"strings"
+
+	"github.com/spf13/cobra"
+)
+
+var posArgRegex = regexp.MustCompile(`\[.*?]|\(.*?\)|--.*?<.*?>|<(.*?)>`) // matches all positional args in group 1
+
+func ValidateExact(cmd *cobra.Command, args []string) error {
+	return validate(cmd, args, true)
+}
+
+func Validate(cmd *cobra.Command, args []string) error {
+	return validate(cmd, args, false)
+}
```

Comment:
Could you add some doc strings to these functions to explain what they do and how they differ?

In addition it would be nice to have this in the `CONTRIBUTING.md`.